projects/aca-content/src/lib/components/view-profile/view-profile.component.html

```diff
@@ -49,104 +78,156 @@ <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.LAST_NAME'
+            <input type="tel" value="" formControlName="mobile" class="app-profile-general-dropdown-input-details app-selected">
+          }
+          @if (profileForm.get('mobile').invalid) {
+            <mat-error class="app-error-message">
+              {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
+            </mat-error>
+          }
         </div>
       </div>
-    </div>
+    }
   </div>
 
-  <div class="app-profile-contact-row" [formGroup]="profileForm">
+  <div class="app-profile-contact-row" [formGroup]="profileForm" id="contact-dropdown">
     <div class="app-profile-general profile-general-bottom-radius">
       <div class="app-profile-general-section">
-        <mat-icon class="app-profile-general-icon" (click)="toggleContactDropdown()" id="toggle-contact-dropdown">
-          {{ contactSectionDropdown ?  'expand_more' : 'chevron_right'}}</mat-icon>
+         <button
+          mat-icon-button
+          (click)="toggleContactDropdown()"
+          [attr.title]="`APP.TOOLTIPS.${contactSectionDropdown ? 'COLLAPSE' : 'EXPAND'}_SECTION` | translate"
+          [attr.aria-label]="`APP.TOOLTIPS.${contactSectionDropdown ? 'COLLAPSE' : 'EXPAND'}_SECTION` | translate">
+          <mat-icon>{{ contactSectionDropdown ?  'expand_more' : 'chevron_right'}}</mat-icon>
+        </button>
         <h4 class="app-general-title">{{'APP.EDIT_PROFILE.COMPANY_DETAILS' | translate}}</h4>
       </div>
-
-      <div class="app-general-edit-btn" *ngIf="contactSectionButtonsToggle">
-        <button mat-raised-button class="app-general-edit" id="contact-edit-button"
-          (click)="toggleContactButtons()">{{'APP.EDIT_PROFILE.EDIT' | translate}}</button>
-      </div>
-      <div class="app-general-edit-btn" *ngIf="!contactSectionButtonsToggle">
-        <button mat-raised-button class="app-general-cancel-btn" id="contact-cancel-button"
-          (click)="toggleContactButtons()">{{'APP.EDIT_PROFILE.CANCEL' | translate}}</button>
-        <button mat-raised-button  class="app-general-save-btn" id="contact-save-button" [disabled]="profileForm.invalid"
-          (click)="onSaveCompanyData(profileForm)">{{'APP.EDIT_PROFILE.SAVE' | translate}}</button>
-      </div>
+      @if (contactSectionButtonsToggle) {
+        <button
+          mat-icon-button
+          (click)="toggleContactButtons()"
+          class="app-general-edit"
+          [attr.title]="'APP.TOOLTIPS.EDIT' | translate"
+          [attr.aria-label]="'APP.TOOLTIPS.EDIT' | translate">
+          <mat-icon>edit</mat-icon>
+        </button>
+      } @else {
+        <button
+          mat-icon-button
+          class="app-general-cancel-btn"
+          [attr.title]="'APP.TOOLTIPS.CANCEL' | translate"
+          [attr.aria-label]="'APP.TOOLTIPS.CANCEL' | translate"
+          (click)="toggleContactButtons()">
+          <mat-icon>close</mat-icon>
+        </button>
+        <button
+          mat-icon-button
+          color="primary"
+          class="app-general-save-btn"
```

Comment:
I can't see usage of that class - if that is for testing only then we should to use data-automation-id instead